showcase/src/app/news/news-item-component/news-item.component.html

```diff
@@ -1,12 +1,12 @@
-<mat-card class="news-item">
-	<div class="mat-card-title">
-		<h2><ng-content select=".news-item-title"></ng-content></h2>
-	</div>
-	<div class="mat-card-subtitle">
-		<h4>Release: {{ release }} <br>Date: {{ newsDate }}
+<div class="border-line">
+	<h2><ng-content select=".news-item-title"></ng-content></h2>
+	
+	<div>
+		<h4 class="information">Release: {{ release }} ({{ newsDate }})
 		</h4>
 	</div>
-	<div class="mat-card-content">
+	<div><ng-content class="image" select=".news-image"></ng-content></div>
+	<div class="mat-body-2">
```

Comment:
I think it is better to remove the `mat-body-2` from this content because it is all bold and it looks strange. We could use the `mat-body-2` class in the subtitle "Release xxx (xx/xx/xxxx)". What do you think?